Cloth.Application/Interfaces/Repositories/IClothSizeRepository.cs

```diff
@@ -1,7 +1,7 @@
 ﻿namespace Cloth.Application.Interfaces.Repositories;
 
 using Cloth.Domain.Entities;
-using Persistence.Abstractions.Interfaces;
+using PersistenceAbstractionsInterfaces;
```

Comment:
This is not Standard, put it back the way it was

Reply:
Assuming this is the conflicting namespace:
```
using PersistenceAbstractions1 = Persistance.Abstractions.Interfaces;
using PersistenceAbstractions2 = Persistance.Abstractions.Interfaces; // 
```

Now you can use the aliased namespaces to differentiate between the two classes:

```
PersistenceAbstractions1.YourClass
PersistenceAbstractions2.YourClass
```


https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/using-directive#using-alias